components-ui/horizontal-separator/index.tsx

```diff
@@ -17,6 +17,7 @@ export const SimpleSeparator = () => (
 
 export const HorizontalSeparator = () => (
   <>
+    <BreakPageForPrint />
     <div className="horizontal-separator layout-center">
       <span className="line" />
       <span className="circle" />
```

Comment:
This might create empty blank page where we called `<HorizontalSeparator/>`next to `<BreakPageForPrint />`
Also, we sometimes use several separator pretty closelys.

But I like the idea. If we dfeploy this, I would use a `<PrintNever />` on the separator itself as it is no longer required